lib/govuk_taxonomy_helpers/publishing_api_response.rb

```diff
@@ -42,16 +29,25 @@ def initialize(content_item:, expanded_links:)
         base_path: content_item["base_path"]
       )
 
-      add_expanded_links(expanded_links)
+      add_expanded_links(expanded_links, publishing_api)
     end
 
   private
 
-    def add_expanded_links(expanded_links_response)
+    def add_expanded_links(expanded_links_response, publishing_api)
+      level_one_taxons = expanded_links_response["expanded_links"]["level_one_taxons"]
       child_taxons = expanded_links_response["expanded_links"]["child_taxons"]
       parent_taxons = expanded_links_response["expanded_links"]["parent_taxons"]
       taxons = expanded_links_response["expanded_links"]["taxons"]
 
+      if !level_one_taxons.nil?
```

Comment:
To make this statement easier to read you can make this test truthy, by checking if level_one_taxons exist.

e.g. `if level_one_taxons`